src/main/java/core/basesyntax/utils/generate/impl/csv/CsvGeneratorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.utils.generate.impl.csv;
+
+import core.basesyntax.storage.impl.StorageImpl;
+import core.basesyntax.utils.generate.ReportGenerator;
+
+public class CsvGeneratorImpl implements ReportGenerator {
+    private static final String LINE_SEPARATOR = ",";
```

Comment:
The name `LINE_SEPARATOR` is a bit misleading because it's commonly used to refer to a newline character or sequence. Consider renaming it to `COLUMN_SEPARATOR` or `FIELD_SEPARATOR` to more accurately describe its purpose.